hera_pspec/pspecdata.py

```diff
@@ -3357,6 +3435,13 @@ def pspec(self, bls1, bls2, dsets, pols, n_dlys=None,
             given an input visibility noise model, and store the output
             in the UVPSpec object.
 
+        store_QE_error : bool, optional
```

Comment:
can we change this to `store_cov_diag`, which I think makes clearer what is actually happening. The doc string is fine, I think

Reply:
Sure.